docs/conf.py

```diff
@@ -195,25 +186,185 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+try:
+    import nvidia_sphinx_theme  # noqa: F401
+
+    html_theme = "nvidia_sphinx_theme"
+except ImportError:
+    import sphinx_rtd_theme
+
+    html_theme = "sphinx_rtd_theme"
+    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+
+
+html_theme_options = {
+    "switcher": {
+        # use for local testing
+        # "json_url": "http://localhost:8888/_static/switcher.json",
+        "json_url": "https://docs.nvidia.com/deeplearning/dali/user-guide/"
+        "docs/_static/switcher.json",
```

Comment:
I wonder if there's an option (or a hack) to make this URL relative to the base path of the docs. Then you wouldn't have to change the URL to test locally. I'll research this

Reply:
> I wonder if there's an option (or a hack) to make this URL relative to the base path of the docs.

If so I would do this only to some extend. We don't want the dev-documentation to use own switcher. We want it to refer to the global one.

Reply:
I see the PyData theme authors define this during the generation - https://github.com/pydata/pydata-sphinx-theme/blob/main/docs/conf.py#L112 and https://github.com/pydata/pydata-sphinx-theme/blob/main/docs/conf.py#L126.

Reply:
Nice! We can do the same but I'm fine with doing it as a follow-up later - it's a minor thing and not user-facing

Reply:
I'm not sure if we can. We don't have a differentiator between nightly/weekly/official builds/CI builds.

Reply:
Ah, that's right. Let's keep it as it is, having this dropdown in dev docs is not worth it ;)